### ISSUE TITLE
Feature requests - add to SAR

### ISSUE THREAD

**Issue by mikebroberts** [rhboyd/CustomResourceACM#2]
Thanks - this worked!

I'm not a frequent Python developer, so I had a little trouble packaging the CR on my mac (https://stackoverflow.com/questions/24257803/distutilsoptionerror-must-supply-either-home-or-prefix-exec-prefix-not-both saved the day) . 

It would be awesome if the CR was available as a public SAR app so that I didn't need to worry about packaging it myself.

Thanks again!

**Comment by rhboyd** [comment]
I completely feel your pain. I was doing a Twitch Stream last night and I spent 20 minutes on the exact same issue. My biggest concerns about putting this in SAR are (1) I would need to properly implement the update/delete methods to properly update/delete the cert when the stack is modified, (2) correctly scope the lambda permissions to the specific resources involved, and (3) ACM certs are super important to securing cloud resources and I am not trying to get an angry letter from some rando lawyer because someone deployed my SAR app and ended up using their acm cert to sign malware or some nonsense.

**Comment by mikebroberts** [comment]
Oh, yeah, that's a fair call. Lawyers! :)

Again, I'm not a python expert at all, but might using `sam build` help the dependencies problem? https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/sam-cli-command-reference-sam-build.html

**Comment by rhboyd** [comment]
yeah, sam build should resolve the dependencies problem. I can update the readme to include steps for running this via sam build to get the container isolation.
